oauth2client/pkce.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
> I thought about that, but client.py is already way too long, IMHO. If you want me to inline it, I will, but I think we don't want to be adding still more stuff to it.

I agree, but also these are literally one-line functions. I'm not sure if the indirection is worth it. @nathanielmanistaatgoogle what do you think?

> I don't feel strongly about it, but somebody might want to use those functions in some other context. For instance, as part of a server implementation of PKCE. It's not like these are complex internals that we want to discourage users from tinkering with.

We are desperately trying to reduce the public API surface of this library so that it is easier for us to improve it. If we do decide to keep this as a separate module, please do make it private.